packages/types/types/index.d.ts

```diff
@@ -74,6 +85,24 @@ export interface HideProps extends BoxProps {
   print?: boolean
 }
 
+export interface IconProps
+  extends ColorProps,
+  SizeProps {
+  name?: string
+  title?: string
+  titleId?: string
+  desc?: string
+  descId?: string
+}
+
+export interface IconButtonProps extends ButtonProps {
+  icon?: React.ComponentElement<IconProps, any>
+  onClick?: Function
+  dsRef?: RefPropType
+}
+
+export interface IconFieldProps extends FlexProps {}
```

Comment:
Can you try running `npm run test` from the types project? I think this will complain about `no-empty-interface`. In this case, you should just be able to use `FlexProps` as the prop types for `IconField` since they use the same interface.

Reply:
Yup you were right, fixed!